backend/Wheelingful.Services/Extensions/BookManagerExtension.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Wheelingful.Core.Contracts.Books;
+
+namespace Wheelingful.Services.Extensions;
+
+public static class BookManagerExtension
+{
+    public static async Task<bool> IsActualAuthor(this IBookManager manager, int bookId, string userId)
```

Comment:
should this method be a part of IBookManager?

Reply:
Maybe, maybe not. There is no point to refactor it before the next sprint anyway. I plan on architecture overhaul.